grails-app/domain/org/pih/warehouse/core/Location.groovy

```diff
@@ -57,8 +58,9 @@ class Location implements Comparable<Location>, java.io.Serializable {
     Date dateCreated
     Date lastUpdated
 
-    static belongsTo = [parentLocation: Location, organization: Organization]
+    static belongsTo = [parentLocation: Location, organization: Organization, zone: Location]
```

Comment:
I don't think we need this. We just want the zone_id to be added to the location table. It's the same as location group and will only be used to group bin locations by zone.